### PR TITLE
[opentitantool] Move load-bitstream and set-pll into a new fpga command group.

### PR DESCRIPTION
**Pull request by jrvanwhy**
I am grouping these commands in preparation of adding a third FPGA command that resets the board (issue #14686).

### CODE REVIEW

**Comment by cfrantz** [comment]
LGTM, but:
- You need to amend your commit to have `Signed-off-by: <foo>`.
- You need to update the bazel automation that loads bitstreams to reflect the new command hierarchy.
- You need to update the various documentation that refers to `opentitantool load-bitstream` to reflect the new command hierarchy.

**Review by alphan — APPROVED** [review]
Thanks @jrvanwhy! No additional comments besides @cfrantz's feedback.

**Comment by jrvanwhy** [comment]
Please disregard my push; I'm still working on this PR.

**Comment by jrvanwhy** [comment]
> LGTM, but:
> 
> * You need to amend your commit to have `Signed-off-by: <foo>`.
> * You need to update the bazel automation that loads bitstreams to reflect the new command hierarchy.
> * You need to update the various documentation that refers to `opentitantool load-bitstream` to reflect the new command hierarchy.

All done, and I've run as many tests as I can on my system.

File: sw/host/opentitantool/src/main.rs [hunk 1]
```diff
@@ -28,11 +28,10 @@ enum RootCommandHierarchy {
     Gpio(command::gpio::GpioCommand),
     Emulator(command::emulator::EmuCommand),
 
+    Fpga(command::fpga::FpgaCommand),
     I2c(command::i2c::I2cCommand),
     Image(command::image::Image),
-    SetPll(command::set_pll::SetPll),
     UpdateUsrAccess(command::update_usr_access::UpdateUsrAccess),
```
Comment (alphan) [inline sw/host/opentitantool/src/main.rs:34]: nit: would you mind moving also this under the new command and updating references?

**Comment by alphan** [comment]
@moidx is the kokoro failure blocking? I checked the results and it just shows a successful lint job.

**Comment by moidx** [comment]
@alphan all good now.

**Comment by alphan** [comment]
> @alphan all good now.

Thanks @moidx !

**Comment by alphan** [comment]
@jrvanwhy I'll merge this now. We can address `UpdateUsrAccess` in a separate PR.

File: sw/host/opentitantool/src/main.rs (hunk 1 above)
Comment (jrvanwhy) [reply sw/host/opentitantool/src/main.rs:34]: As I understand it, implementing the reset command is a higher priority, so I filed #14950 to track moving `update-usr-access` under the `fpga` command.

**Comment by cfrantz** [comment]
Post submit LGTM.

**Pull request merged**